app/assets/javascripts/account/AccountController.js.coffee

```diff
@@ -8,7 +8,8 @@ AccountController = (
   ShortFormApplicationService,
   SharedService,
   ModalService,
-  inputMaxLength
+  inputMaxLength,
+  ListingIdentityService
```

Comment:
sorry to horn in on your review @akegan, but @software-project could you group the inclusion of the services together and also put the service inclusions in alphabetical order?